package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "corona-warn-app-landingpage",
-  "version": "1.1.0",
+  "version": "1.2.0",
   "description": "Corona-Warn-App website coronawarn.app",
   "main": "gulpfile.js",
```

Comment:
@MikeMcC399 

Since gulpfile.js was renamed to gulpfile.mjs, I think it would be good to rename it here as well. If I have grasped this correctly, this isn't changing anything in functionality and is just for informative purposes, but it would still be good to have the correct file extension in there if we're already making changes to the file

Reply:
@larswmh 

Good catch! I don't believe this is used anywhere but I have corrected it anyway in https://github.com/corona-warn-app/cwa-website/pull/3309/commits/d78aa2d1b2a6ae69daedb5cedb68665b87009bd4.